sdk/log/provider.go

```diff
@@ -30,6 +29,7 @@ const (
 type providerConfig struct {
 	resource      *resource.Resource
 	processors    []Processor
+	filterers     []Filterer
```

Comment:
Just like the `sampler` in `TracerProvider`, could we only have a single instance here? If no one is complaining about the current implementation of the `sampler`, maybe it is ok to have just one instance.

Reply:
I think this approach is simpler and more consistent with how processing works.

For reference, the samplers can be decorators e.g. [ParentBased Sampler](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#parentbased). There is nothing technically preventing creating a filterer decorator. However, I feel that accepting multiple filterers increases the ease of use.